code/japanese/reading.py

```diff
@@ -152,29 +152,30 @@ def reading(self, expr: str) -> str:
                 if kanji[i] != reading[i]:
                     break
                 placeL = i + 1
-            if placeL == 0:
-                if placeR == 0:
```

Comment:
As placeL/R are not optional, I think comparing to 0 is a little more explicit. Not a big deal though.

Reply:
`not x` is more pythonic than the more explicit `x == 0`. I'll revert it if you wish.